### ISSUE TITLE
Requirement 11 is hard to read

### ISSUE THREAD

**Issue by jyutzler** [opengeospatial/geopackage#110]
From Michael Gerlek http://www.slideshare.net/mpgx/geo-package-pointclouds slide 15

> A bit hard to parse – suggest “record” clauses be separated by semicolons

**Comment by jyutzler** [comment]
At today's SWG we agreed that this was hard to read. We will add a table showing the three required records.

**Comment by jyutzler** [comment]
Please take a look at http://www.geopackage.org/spec/#_requirement-11 and let me know if this is better.

**Comment by mpgerlek** [comment]
Much better, thank you.
